lib/modules/manager/devcontainer/readme.md

```diff
@@ -0,0 +1,3 @@
+Extracts all Docker images from the `image` and `features` properties.
+
+Does not support `build.dockerFile` or `dockerComposeFile` values as these are covered by existing managers.
```

Comment:
Should we link to the other managers that cover those fields? 

I guess the `docker` and `docker-compose` managers deal with those fields?